### PR TITLE
 Make indent CI check abort on Doxygen warnings

### PR DESCRIPTION
**Pull request by masterleinad**
Apparently, this wasn't working correctly.

### CODE REVIEW

**Comment by masterleinad** [comment]
OK. This seems to finally work.

**Comment by drwells** [comment]
Looks good. Why did you only need to convert some of the `@deprecated` messages?

**Comment by masterleinad** [comment]
> Looks good. Why did you only need to convert some of the `@deprecated` messages?

I couldn't quite figure that out. The output before the last commit was
```
deprecated:274: warning: explicit link request to 'isotropic_refinement' could not be resolved
deprecated:278: warning: explicit link request to 'isotropic_refinement' could not be resolved
deprecated:282: warning: explicit link request to 'isotropic_refinement' could not be resolved
```
Looking at `deprecated.html` all of them were related to deprecations in `qprojector.h` so I changed only the ones in that file.

File: examples/step-69/step-69.cc [hunk 1]
```diff
@@ -2183,7 +2183,7 @@ namespace Step69
   // The second thing to note is that we have to compute global minimum and
   // maximum $\max_j |\nabla r_j|$ and $\min_j |\nabla r_j|$. Following the
   // same ideas used to compute the time step size in the class member
-  // <code>%TimeStepping<dim>::step()</code> we define $\max_j |\nabla r_j|$
+  // <code>%TimeStepping\<dim>::%step()</code> we define $\max_j |\nabla r_j|$
```
Comment (tjhei) [inline examples/step-69/step-69.cc:2186]: is the ``%`` still necessary?

Comment (masterleinad) [reply examples/step-69/step-69.cc:2186]: Yes, it was.

**Review by tjhei — APPROVED** [review]

**Comment by tjhei** [comment]
Thanks, @masterleinad . This will make doxygen fixes before each release so much easier for me. :+1:

**Pull request merged**